frontend/src/components/bucket/BucketList.vue

```diff
@@ -17,9 +17,7 @@ const closeInfo = () => {
   displayInfo.value = null;
 };
 
-onMounted(() => {
-  useToaster(bucketStore.load, { summary: 'Unable to load buckets.' });
-});
+await useToaster(bucketStore.load, { summary: 'Unable to load buckets.' });
```

Comment:
Think this was just an experiment while screen sharing? Could keep back in onMounted for now